arduino/sketch_read_transmit_from_sd.ino

```diff
@@ -0,0 +1,60 @@
+#include <SD.h>  // Load SD library
+#include <SPI.h>
+
+int chipSelect = 4;  // chip select pin for the MicroSD Card Adapter
+File file;  // File object that is used to read and write data
+
+/*
+ * Reads from SD card and transmit it over the serial output channel line by line with '\n' as the line delimitter. Skips null lines.
+ */
+void readFromSDCardToSerialOutputLineByLine(){    // delay value can be passed in as an argument
```

Comment:
Let's break this function in to smaller modules. Makes easier to debug. One func to read from SD card, one to decode etc..